dotnet/Directory.Packages.props

```diff
@@ -103,7 +103,7 @@
     <PackageVersion Include="DuckDB.NET.Data" Version="1.1.2.1" />
     <PackageVersion Include="MongoDB.Driver" Version="2.30.0" />
     <PackageVersion Include="Microsoft.Graph" Version="4.51.0" />
-    <PackageVersion Include="Microsoft.Identity.Client.Extensions.Msal" Version="2.28.0" />
```

Comment:
Major version bump, so I want to investigate where this is being used

Reply:
Used by
- Microsoft.Identity.Client.Extensions.Msal
- Microsoft.SemanticKernel.Plugins.OpenApi

I want to test both of these before taking this update